pkgs/development/python-modules/sip/fix-manylinux-version.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/sipbuild/project.py b/sipbuild/project.py
+--- a/sipbuild/project.py
++++ b/sipbuild/project.py
+@@ -336,13 +336,13 @@ class Project(AbstractProject, Configurable):
+             # We expect a two part tag so leave anything else unchanged.
+             parts = platform_tag.split('-')
+             if len(parts) == 2:
+-                if self.minimum_glibc_version > (2, 17):
++                if self.minimum_glibc_version > (2, 17) or parts[1] not in {"x86_64", "i686", "aarch64", "armv7l", "ppc64", "ppc64le", "s390x"}:
```

Comment:
Thank you so much for writing this PR!

I think `ppc64le` here (which is a Debian-specific identifier, like `amd64`) is supposed to be `powerpc64le`.  When I built on a powerpc64le machine with this PR I got the same error I was getting on aarch64 without it, except that it mentioned `powerpc64le`.

I changed it and am verifying with a rebuild, but it'll take a while.

`ppc64` might need to be changed as well but I don't have hardware to test that change on.